build/libs/pom.xml

```diff
@@ -58,6 +58,11 @@
       <artifactId>org.spoofax.jsglr</artifactId>
       <version>${metaborg-version}</version>
     </dependency>
+    <dependency>
```

Comment:
I don't think JSGLR2 needs to go here, unless Stratego/XT needs it, but that still uses the old JSGLR.

Reply:
I think you are right. Removed in 8bb7c8df77f356f42572. Verified with running `./b build all`.